c7n/resources/glue.py

```diff
@@ -272,3 +272,122 @@ def process(self, resources):
                 client.delete_table(DatabaseName=r['DatabaseName'], Name=r['Name'])
             except client.exceptions.EntityNotFoundException:
                 continue
+
+
+@resources.register('glue-classifier')
+class GlueClassifier(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'glue'
+        enum_spec = ('get_classifiers', 'Classifiers', None)
+        id = name = 'Name'
+        date = 'CreationTime'
+        arn_type = 'classifier'
+
+    permissions = ('glue:GetClassifiers',)
+
+
+@GlueClassifier.action_registry.register('delete')
+class DeleteClassifier(BaseAction):
+
+    schema = type_schema('delete')
+    permissions = ('glue:DeleteClassifier',)
+
+    def process(self, resources):
+        client = local_session(self.manager.session_factory).client('glue')
+        for r in resources:
+            classifier = next(iter(r.values()))
```

Comment:
this is quite odd

Reply:
I changed it to `classifier = list(r.values())[0]`. I need to get the value of a single item dictionary, so hopefully this is more readable.

Reply:
what are the attributes of a resource here?, this reads like grab a random attribute of a resource.

Reply:
reading through thee docs, its apparent. it would be useful when something is odd to document inline, ie a reference to the resource attributes https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/glue.html#Glue.Client.get_classifier